.eslintrc.js

```diff
@@ -4,10 +4,20 @@ module.exports = {
     ecmaVersion: 6,
     sourceType: 'module'
   },
-  extends: 'eslint:recommended',
+  extends: [
+    'eslint:recommended',
+    'prettier'
+  ],
+  plugins: [
+    'prettier'
+  ],
   env: {
     browser: true
   },
   rules: {
+    'prettier/prettier': ['error', {
```

Comment:
I kept the same 120-width setting that we agreed on earlier, and also set it up to prefer single quotes, which I think it a better style for JS code. My preference is single quotes for JS and within handlebars and double quotes within HTML.